homeassistant/components/qnap/sensor.py

```diff
@@ -190,106 +176,90 @@
     ),
 )
 
-SENSOR_KEYS: list[str] = [
-    desc.key
-    for desc in (
-        *_SYSTEM_MON_COND,
-        *_CPU_MON_COND,
-        *_MEMORY_MON_COND,
-        *_NETWORK_MON_COND,
-        *_DRIVE_MON_COND,
-        *_VOLUME_MON_COND,
-    )
-]
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
You cannot drop the `PLATFORM_SCHEMA`.
It is required to keep it during the YAML deprecation period.